static/css/tools/base_tools.scss

```diff
@@ -44,7 +45,7 @@ body {
 #main-footer #sub-footer {
   border-top: none !important;
   margin-bottom: 0 !important;
-  padding-bottom: 1.5rem !important;
+  padding: 8px 0px !important;
```

Comment:
I don't think this will be needed anymore when my PR goes in

Reply:
Ack, this is now reflected after merging with current master.